syntax_and_semantics/literals/string.md

```diff
@@ -175,28 +175,23 @@ Leading whitespace is removed from the heredoc contents according to the number
 <<-STRING
   Hello
     world
-  STRING # => "Hello\n  world"
+  STRING
+# => "Hello\n  world"
 
 <<-STRING
     Hello
       world
-  STRING # => "  Hello\n    world"
+  STRING
+# => "  Hello\n    world"
 ```
 
-It is possible to directly call methods on heredoc string literals, or use them inside parentheses:
+It is possible to directly call methods on heredoc string literals:
 
 ```crystal
-<<-SOME
+<<-SOME.upcase
 hello
-SOME.upcase # => "HELLO"
-
-def upcase(string)
-  string.upcase
-end
-
-upcase(<<-SOME
-  hello
-  SOME) # => "HELLO"
+SOME
+# => "HELLO"
```

Comment:
The result should be presented at the first line (`<<-SOME.upcase # => "HELLO"`)